src/test_pages/TimelinePage.ts

```diff
@@ -0,0 +1,45 @@
+import { Locator, Page, expect } from '@playwright/test'
+import { BasePage } from './BasePage'
+
+export default class TimelinePage extends BasePage {
+  private date: Locator
+  private operator: Locator
+  private operators_dropdown: Locator
+  private operators_list: Locator
+  private line_number: Locator
+  private close_line_number: Locator
+  private route_select: Locator
+  private station_select: Locator
+  private eged_taavura: Locator
+
+  constructor(protected page: Page) {
+    super(page)
+    this.date = this.page.locator("//input[@placeholder='DD/MM/YYYY']")
+    this.operator = this.page.locator('#operator-select')
+    this.line_number = this.page.locator("//input[@placeholder='לדוגמא: 17א']")
+    this.close_line_number = this.page.locator("span[aria-label='close']")
+    this.route_select = this.page.locator('#route-select')
+    this.operators_dropdown = this.page.locator("button[aria-label='Open']")
+    this.operators_list = this.page.locator('ul#operator-select-listbox')
+    this.eged_taavura = this.page.locator("//li[text()='אגד תעבורה']")
+    this.station_select = this.page.locator('')
+  }
+
+  public async selectOperatorFromDropbox(operatorName: string) {
+    await this.selectFrom_UL_LI_Dropbox(this.operators_dropdown, this.operators_list, operatorName)
+  }
+
+  public async fillLineNumber(lineNumber: string) {
+    await this.fillTextToElement(this.line_number, lineNumber)
+  }
+
+  public async closeLineNumber() {
+    await this.clickOnElement(this.close_line_number)
```

Comment:
don't you agree that
```suggestion
    await this.close_line_number.click()
```
will be simpler?

Reply:
additionally, using `page.locator("span[aria-label='close']").click()` let the user immediately see and understand which selector is used. using `this.close_line_number` is better in terms of reusability, but since we don't reuse this thing in many places, I think it's a little הכנה למזגן at this stage

Reply:
I can't agree with that because inside this.clickOnElement it's wrapped with test.step that can help with debugging, let's leave it as it.

Reply:
> additionally, using `page.locator("span[aria-label='close']").click()` let the user immediately see and understand which selector is used. using `this.close_line_number` is better in terms of reusability, but since we don't reuse this thing in many places, I think it's a little הכנה למזגן at this stage

This is the difference between somebody who is not working as an Automation developer but knows all different aspects of it and even can write automation by itself. I'll explain why I tell this )) Writing tests it's like writing a story. If you want to learn the system, learn it from automation tests. And when somebody new look on your test he should understand on the spot what this locator means (because it takes second to press ctrl + mouth click). When you fix your own test, you should know immediately what this locator means, but not start searching the css or xpath or whatever in browser. And as well your code is more organized like this. All locators in the same place, all functions and getters in other.

Reply:
I don't understand your comment.
I don't "knows all different aspects of it", and so do you.
I prefer simple solutions rather than smart ones.
We removed Yarn recently, not because it wasn't the right solution for the project. We did it because it's the right solution for us.
We have many junior contributors, and I don't find the complexity justified in this case.

Reply:
As you see in the ticket https://github.com/hasadna/open-bus-map-search/issues/296 I've planned dozens of tests to be done on this page, and I hope the functionality allowing to plan more additional tests. I hope the solution of POM is not the to smart ones, but the needed one. But of course I will be more than happy if junior contributors will learn to understand the difference between the main idea of using it or not.